apps/studio/src/features/editing-experience/components/ComplexEditorStateDrawer.tsx

```diff
@@ -1,12 +1,48 @@
-import { Box } from "@chakra-ui/react"
+import { Box, Heading, HStack, Icon } from "@chakra-ui/react"
+import { IconButton } from "@opengovsg/design-system-react"
+import { BiDollar, BiX } from "react-icons/bi"
 
 import FormBuilder from "./form-builder/FormBuilder"
 
 export default function ComplexEditorStateDrawer(): JSX.Element {
   return (
-    <Box p={4}>
-      <h1>Complex Editor State Drawer</h1>
-      <FormBuilder component="infocards" />
+    <Box position="relative" h="100%" w="100%">
+      <Box
+        bgColor="base.canvas.default"
+        borderBottomColor="base.divider.medium"
+        borderBottomWidth="1px"
+        px="2rem"
+        py="1.25rem"
+      >
+        <HStack justifyContent="space-between" w="100%">
+          <HStack spacing={3}>
+            <Icon
+              as={BiDollar}
+              fontSize="1.5rem"
+              p="0.25rem"
+              bgColor="slate.100"
+              textColor="blue.600"
+              borderRadius="base"
+            />
+            <Heading as="h3" size="sm" textStyle="h5" fontWeight="semibold">
+              {/* TODO: Replace this with the actual component name */}
+              Edit Infocols
+            </Heading>
+          </HStack>
+          <IconButton
+            icon={<Icon as={BiX} />}
+            variant="clear"
+            colorScheme="sub"
+            size="sm"
+            p="0.625rem"
+            onClick={() => console.log("Close drawer")}
```

Comment:
Is this a todo?

Reply:
Ah yup, that is done in a future PR where the page states are all wired up together.